pg_duckdb-motherduck/benchmark.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+#sudo apt-get update
+#sudo apt-get install -y docker.io
+#sudo apt-get install -y postgresql-client
+
+# Ubuntu:
+# snap install docker
+# sudo apt install posgresql-client-common
+# sudo apt install postgresql-client-16
+
+# Note: To get equivalent performance you should be runnign from
+# AWS US-EAST-1 region or as close to there as possible. Otherwise
+# you'll see additional latency.
+
+# Sign up for MotherDuck. 
+# Go to the web ui and obtain a token
+# https://motherduck.com/docs/key-tasks/authenticating-and-connecting-to-motherduck/authenticating-to-motherduck/
+# Save the token as the motherduck_token environment variable:
+# export motherduck_token=...
+# create a database called pgclick in the motherduck UI or duckdb cli
+# `CREATE DATABASE pgclick`
+
+if [ -z "${MOTHERDUCK_TOKEN}" ]; then
+    echo "Error: MOTHERDUCK_TOKEN is not set."
+    exit 1
+fi
+
+sudo docker run -d --name pgduck --network=host -e POSTGRES_PASSWORD=duckdb -e MOTHERDUCK_TOKEN=${MOTHERDUCK_TOKEN} pgduckdb/pgduckdb:16-main -c duckdb.motherduck_enabled=true
+
+# Give postgres time to start running
+sleep 10 
+
+./load.sh 2>&1 | tee load_log.txt
```

Comment:
I tried to run `benchmark.sh` (after getting a token from Motherduck) but ran into an error

```
psql: error: connection to server at "localhost" (127.0.0.1), port 5432 failed: FATAL:  password authentication failed for user "postgres"
connection to server at "localhost" (127.0.0.1), port 5432 failed: FATAL:  password authentication failed for user "postgres"
```

during load (l. 36). Don't we need to specify a password in l. 31 as well?

Reply:
Hi @rschu1ze ... is there any chance you already had a postgres instance running? The password is provided in load.sh in the connection string:
CONNECTION=postgres://postgres:duckdb@localhost:5432/postgres
Which is the same password we set when creating the instance. Can you run psql to connect to your local instance? What connection string do you need to provide?

I reran this on a clean ubuntu instance on ec2 and it executed with error.

Reply:
Nevermind, everything now worked for me, thanks.